### PR TITLE
Minor edits

### PR DESCRIPTION
**Pull request by iadawn**

### CODE REVIEW

File: content/no-code.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ The templates, components, and pre-authored content that your tool provides need
 ### The tool helps make images and videos accessible
 
 <p><span class="persona">Persona: </span>
-  Dacso runs an automotive repair shop and he does a lot of body work. His daughter created a website for him to showcase his work. Dacso uploads before-and-after photos of his projects. He also posts videos with simple maintenance tips.
+  Dacso runs an car repair shop and does a lot of body work. His daughter created a website for him to showcase his work. Dacso uploads before-and-after photos of his projects. He also posts videos with simple maintenance tips.
```
Comment (daniel-montalvo) [inline content/no-code.md:152]: "Mechanics",, "automotive" "car repair" I don't feel strongly about any of these. Shawn suggested "automotive repair"

Comment (daniel-montalvo) [inline content/no-code.md:152]:
```suggestion
  Dacso runs a car repair shop and does a lot of body work. His daughter created a website for him to showcase his work. Dacso uploads before-and-after photos of his projects. He also posts videos with simple maintenance tips.
```

File: content/social-media.md [hunk 2]
```diff
@@ -148,18 +148,18 @@ People who use social media have different abilities. For example, they may read
...
     <li>
       <p><span class="issue">Problem: </span><span><q>
-        My client has now requested me to make all media accessible, including images, audio, and video. I am not sure how to get this done, and I can’t find anything on my tool that explains it.
+        My client has now requested that I make all media accessible, including images, audio, and video. I am not sure how to get this done, and I can’t find anything in my tool that explains it.
       </q></span></p>
     </li>
     <li>
       <p><span class="issue">Works well: </span><span><q>
-        When I upload an image, the tool asks me if it needs alternative text and gives guidance on how to write it. When I upload a video, the tool suggests me to add transcripts, captions, and descriptions as appropriate.
+        When I upload an image, the tool asks me if it needs alternative text and gives guidance on how to write it. When I upload a video, the tool suggests that I add transcripts, captions, and descriptions as appropriate.
```
Comment (daniel-montalvo) [inline content/social-media.md:162]: Not sure what's changed in this line.

File: content/no-code.md (hunk 1 above)
Comment (iadawn) [reply content/no-code.md]: "automotive" seems a bit North American to me. Not sure that 'car' is any more global, but it is certainly simpler.

File: content/social-media.md (hunk 2 above)
Comment (iadawn) [reply content/social-media.md:162]: Changed 'suggests me to' to 'suggests that I'. Just a bit of a grammar correction

**Pull request merged**